imports/plugins/core/core/server/publications/collections/packages.js

```diff
@@ -112,10 +119,19 @@ Meteor.publish("Packages", function (shopId) {
       ], Roles.GLOBAL_GROUP))) {
         options = {};
       }
+
+      const query = { shopId: myShopId };
+
+      // This is to ensure only needed Identity-provider-related routes are published
+      // The env can be one of three: "all", "idp-only", "exclude-idp". Default behavior is "all"
```

Comment:
@ticean I added you for comments about the ENV values. Is the naming good, unambiguous etc?